app/src/components/common/form/more_menu/index.tsx

```diff
@@ -38,6 +38,9 @@ const ItemsContainer = styled.div<{ isOpen: boolean }>`
   display: ${({ isOpen }) => (isOpen ? 'block' : 'none')};
   padding: 8px;
   border-radius: 12px;
+  @media (max-width: ${props => props.theme.themeBreakPoints.md}) {
+    right: -123px;
```

Comment:
This is not a good way to fix this issue.

Instead of `right: -123px`, what about using the following one?

```
right: unset;
left: 0;
```

Reply:
@liaojikunwork ohh didn't know it could be done that way! Will change!